app/controllers/api/volume_mappings_controller.rb

```diff
@@ -1,5 +1,5 @@
 module Api
-  class VolumeMappingsController < BaseProviderController
+  class VolumeMappingsController < BaseController
```

Comment:
Hey @tzurvaza just curious why change back to the BaseController here?

Reply:
Interesting that is surprising that it is causing issues as it only adds `OPTIONS` allowing for you to use DDF params defined in your provider cc @kbrock

Reply:
When I use `BaseProviderController` I got error (on the `OPTIONS` request): 
```
{
  "kind": "internal_server_error",
  "message": "undefined method `params_for_create' for #<Class:0x0000559a422769d8>",
  "klass": "NoMethodError"
}
```

Reply:
@agrare It's been a long time without an update and I want to push the changes, any update here?

Reply:
Was waiting for resolution to https://github.com/ManageIQ/manageiq/pull/2197

Reply:
Okay if we aren't subclassing these we don't need params_for_create so i'm good with this for now

Reply:
Sorry I did not reply.
This was added so we could get OPTIONS and ddf working.
Since it only has the one method (to add OPTIONS support), I'm confused why it would need to be removed

Reply:
followup: #1168 fixed a few issues with `params_for_create`. Seems like that PR would have fixed this issue.

Reply:
I don't think it would have because the issue wasn't the parameters the issue was `::VolumeMapping` has no method `params_for_create`

Reply:
future me: we don't need params here because all children implement the same form so no DDF is needed